study-builder/tenants/atcp/rules/Register User in Pepper.js

```diff
@@ -0,0 +1,228 @@
+function (user, context, callback) {
```

Comment:
What's the difference between this file and the one in `tenants/register-user-in-pepper.js`? I think it's better if we just have one copy of this auth0 rule..